helm/templates/deployment-worker.yaml

```diff
@@ -91,6 +91,11 @@ spec:
         - name: tls
           mountPath: /etc/config/tls
         {{- end }}
+        readinessProbe:
```

Comment:
Do we only want to add a readinessProbe? What about liveness?

Reply:
The current goal is to only handle start up